projects/adapter/tests/test_cli.py

```diff
@@ -252,7 +252,7 @@ def test_selection(capfd):
 
 def test_no_run_results(capfd):
     with ProjectTemporaryDirectory() as tmp_dir:
-        shutil.rmtree(os.path.join(tmp_dir, "mockTarget"))
+        shutil.rmtree(os.path.join(tmp_dir, "mockTarget"), ignore_errors=True)
```

Comment:
This seems sketchy! Why do we have to ignore errors?

Reply:
Perhaps I should have deleted the line instead. ProjectTemporaryDirectory() creates the temp directory but without `mockTarget`, yet `rmtree` will try to delete the entire tree assuming `mockTarget` exists, but as it doesn't, an exception is thrown. Unsure why this line was put there in the first place. I added ignore_errors as the rmtree API has been updated; perhaps this is a new python 3.11 behaviour.

Reply:
@squat I've updated this test to check for the directory's existence first.